app/models/reports/repository.rb

```diff
@@ -45,9 +45,13 @@ def no_bp_measure_count
     #
     def cached_query(calculation, &block)
       items = cache_keys(calculation)
-      cached_results = cache.fetch_multi(*items, force: force_cache?) { |item|
-        block.call(item)
-      }
+      cached_results = if force_cache? # fetch_multi does not support force, so we do it manually here
+        cached_results = items.each_with_object({}) { |item, hsh| hsh[item] = block.call(item) }
```

Comment:
Shall we extract this into a helper method that takes a block? We might have to do a `fetch_multi` in more places.

Reply:
No need now, I patched ActiveSupport directly